modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1274,8 +1274,8 @@ public String toDefaultValue(Schema schema) {
      * @param schema property type
      * @return property value
      */
+    @SuppressWarnings("squid:S3923")
```

Comment:
Might want to comment here that the branches are kept for examples, and the repeated `return "null"` is intentional?

Reply:
sure thing